todo_app/templates/index.html

```diff
@@ -12,7 +12,19 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
```

Comment:
Minor, but we could remove this comment now as the code itself is pretty clear at saying the same, and that way this comment won't go out of date when the site inevitably changes!